docs/pages/packages/primitives.mdx

```diff
@@ -10,22 +10,10 @@ See full list is availabe at [Edge Runtime APIs](/features/available-apis).
 
 ## Installation
 
-<Tabs items={['npm', 'yarn', 'pnpm']} storageKey="selected-pkg-manager">
-  <Tab>
-    ```sh
-    npm install @edge-runtime/primitives --save
-    ```
-  </Tab>
-  <Tab>
-    ```sh
-    yarn add @edge-runtime/primitives
-    ```
-  </Tab>
-  <Tab>
-    ```sh
-    pnpm install @edge-runtime/primitives --save
-    ```
-  </Tab>
+<Tabs items={['npm', 'yarn', 'pnpm']} storageKey='selected-pkg-manager'>
+  <Tab>```sh npm install @edge-runtime/primitives --save ```</Tab>
+  <Tab>```sh yarn add @edge-runtime/primitives ```</Tab>
+  <Tab>```sh pnpm install @edge-runtime/primitives --save ```</Tab>
```

Comment:
It has been formatted with Prettier, which is difficult to deal with because prettier is executed at commit time.

Reply:
yea, no problem 🙂